task_test.go

```diff
@@ -678,3 +640,12 @@ func TestWhenDirAttributeItCreatesMissingAndRunsInThatDir(t *testing.T) {
 	// Clean-up after ourselves only if no error.
 	_ = os.Remove(toBeCreated)
 }
+
+func TestDisplaysErrorOnUnsupportedVersion(t *testing.T) {
+	e := task.Executor{
+		Dir:    "testdata/version/v1",
+		Stdout: ioutil.Discard,
+		Stderr: ioutil.Discard,
+	}
+	assert.Error(t, e.Setup())
```

Comment:
We should also assert the output message here, otherwise this test will pass if another error happen.